src/main/MQ2Utilities.cpp

```diff
@@ -73,12 +49,11 @@ static void DebugSpewImpl(bool always, bool logToFile, const char* szFormat, va_
 
 	vsprintf_s(szOutput, theLen, szFormat, vaList);
 
-	strcat_s(szOutput, theLen, "\n");
 	OutputDebugString(szOutput);
 
 	if (logToFile)
 	{
-		LogToFile(szOutput);
+		SPDLOG_DEBUG(szOutput);
```

Comment:
What happens if your message has a `{}` in it?

Reply:
It logs as expected:

> D 2022-04-17 13:15:12.693835 [mq] Lua Initializing version 0.100000
> D 2022-04-17 13:15:12.716880 [mq] MQ2EQBC Module Loaded
> D 2022-04-17 13:15:12.716979 [mq] MQ2EQBC::Initializing version {20.041599}
> D 2022-04-17 13:15:12.739378 [mq] MQ2AdvPath Module Loaded
> D 2022-04-17 13:15:12.739470 [mq] Initializing MQ2AdvPath
> D 2022-04-17 13:15:12.761004 [mq] MQ2Cast Module Loaded

Reply:
He means the literal `{}` not something wrapped in them.

Reply:
what am I missing?

Reply:
Wasn’t sure how it would react given {} is the format placeholder used by fmt

Reply:
Well, it's using `vsprintf_s()`, not fmt, and that part of the code didn't change.  `{}` aren't used by `vsprintf_s`, instead it uses the `%x` parameter formatting similar to `printf()`

Reply:
>  Under the covers, spdlog::default_logger_raw() does use fmt, but only if args are passed, and this code doesn't pass any args, so the fmt call is skipped.

Ah, this is it, then.